<!-- -->
<launch>
  <arg name="role_name" default="ego_vehicle"/>

  <node pkg="carla_localization" type="carla_manual_control.py" name="carla_localization_manual_control_$(arg role_name)" output="screen" required="true">
    <param name="role_name" value="$(arg role_name)"/>
  </node>

</launch>

